src/utils/fetch.js

```diff
@@ -28,13 +28,29 @@
  * @license Simplified BSD License
  */
 
+const typesToken = {
+  string : 's',
+  number : 'i',
+  boolean : 'b',
+  undefined: 'u',
+  object: 'n'
```

Comment:
But its `typeof` gets object!

Reply:
Yeah, but it says "object", but it's a "n", which semantically is a bit strange. Also, this will never be an object, but null.

So I suggest this:
```
    const isNull = val === null;
    if (typeof val === 'object' && !isNull) {
      return encodeQueryData(val, nesting + `${key}.`);
    } else {
      type = typesToken[isNull ? 'null' : typeof val];
```

Reply:
You are right, I did the changes :)